srcpkgs/spotify-player/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'spotify-player'
+pkgname=spotify-player
+version=0.13.1
+revision=1
+archs="x86_64"
+build_style=cargo
+make_install_args="--path spotify_player"
+makedepends="openssl-devel pkg-config alsa-lib-devel dbus-devel"
+depends="openssl alsa-lib"
```

Comment:
not needed  as shlib dependencies are automatically resolved, see `common/shlibs`